examquestionbank/templates/item_bank/author_view_add_custom.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+
+<div class="insert-new-lib-blocks-here"></div>
+<div class="xblock-header-secondary bank-add-section">
+    <button class="btn btn-primary problem-bank-v2-add-button bank-add-btn">
```

Comment:
```suggestion
    <button class="btn btn-primary problem-bank-v2-add-button bank-add-btn" aria-label="Add problems from content library">
```

To have descriptive labels.